### DISCUSSION TITLE
Is there a way to display the markdown markups (#, ##, ### ** * etc) via the theming features ?

### DISCUSSION THREAD

**Discussion by matthieuHenocque** [gonzalezreal/swift-markdown-ui#218] [Q&A]
Hello there !

I'm looking for a way to make the most simple Markdown Editor possible for my own usage, and I wish to only use bold and italic style in the editor, but keep the markups visible.

Is there a way to do that ?

Thank you very much for your work and your time ! Have a nice day !

**Comment by gonzalezreal** [comment]
Hi @matthieuHenocque,

I don't think MarkdownUI is suited for the editor use case. The sole purpose of this library is to render Markdown-formatted text. So, it can be used for an editor's preview feature rather than for the editor itself. Perhaps you'd like to look into Markdown parsers like https://github.com/apple/swift-markdown and build your editor on top.
